src/components/backend-ai-scaling-group-list.ts

```diff
@@ -275,6 +343,27 @@ export default class BackendAIScalingGroupList extends BackendAIPage {
     );
   }
 
+  _schedulerOptionsRenderer() {
+    const scheduler_opts = Object.entries(JSON.parse(this.scalingGroups[this.selectedIndex].scheduler_opts));
+    const _schedulerKeyValueTemplate = (key, value) => {
+      return html`
+      <vaadin-item>
+        <div><strong>${key}</strong></div>
+        <div class="scheduler-option-value">${value}</div>
+      </vaadin-item>`;
+    };
+
```

Comment:
Could we just remove the extra renderer(_schedulerOptionsRenderer) and put all of HTML into render function? I think it's too much to display a single dialog information.